syntax = "proto3";

option java_multiple_files = true;
option java_package = "io.grpc.examples.helloworld";
option java_outer_classname = "HelloWorldProto";
option objc_class_prefix = "HLW";

package internal_api_template_service;

service InternalApiTemplateService {
  rpc InternalApiTemplateRequest (TemplateRequest) returns (stream TemplateReply);
}

message TemplateRequest {
  string name = 1;
}

message TemplateReply {
  string message = 1;
}
